relnotes/unicodefile.deprecation.md

```diff
@@ -0,0 +1,3 @@
+## Deprecated ocean.io.UnicodeFile
+
+This module is not used anywhere, and is not tested nor maintained.
```

Comment:
`nor` requires `neither`, right @gavin-norman-sociomantic ?

Reply:
Strictly speaking, yes. It should say "and is neither tested nor maintained". (What Mathias wrote sounds fine, on a casual level, though.)